jmix-search/search-starter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -1,3 +1,3 @@
 io.jmix.autoconfigure.search.SearchAutoConfiguration
 io.jmix.autoconfigure.search.IndexingQueueProcessingScheduleAutoConfiguration
-io.jmix.autoconfigure.search.EnqueueingSessionProcessingScheduleAutoConfiguration
```

Comment:
It doesn't look like something was really changed. Exclude this.

Reply:
fixed